sig-api-machinery/charter.md

```diff
@@ -51,7 +51,9 @@ N/A
 
 ### Deviations from [sig-governance]
 
-N/A
+#### Subproject Governance
+
+SIG delegates responsibilities related to any individual subproject, such as repository creation and maintenance for that subproject, to the leads of the given subproject.
```

Comment:
is there a formal way to define subproject leads ? Is the process similar to adding names in OWNER file or some other method ?

Reply:
> is there a formal way to define subproject leads ? Is the process similar to adding names in OWNER file or some other method ?

Not yet. There has been discussion in https://github.com/kubernetes/community/issues/2619, https://github.com/kubernetes/community/issues/1673 and a PR was created to add it https://github.com/kubernetes/community/pull/2731 but it's not possible to formally "define" subproject leads today.

cc @spiffxp @justaugustus

Reply:
I believe this means the people in the owners files listed under the subprojects. If that is an inappropriately large set of people then we need to define that somehow before this can work, I guess.

Reply:
could be an OWNERS_ALIAS in the mean time, if the union of root OWNERs isn't acceptable

Reply:
Can .we rename this subproject owners and move forward.  SIGs *already* delegate responsibility to subproject owners.

From [sig governance](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#subproject-owner).

> * Subproject Owners
>   * SHOULD be an escalation point for technical discussions and decisions in the subproject
>   * SHOULD set milestone priorities or delegate this responsibility

So these are already roles with responsibilities for the subproject.